CommandLine/libEGM/action.cpp

```diff
@@ -116,6 +116,8 @@ std::string Actions2Code(const std::vector< buffers::resources::Action >& action
       case ActionKind::ACT_NORMAL:
         if (action.exe_type() == ActionExecution::EXEC_NONE) break;
 
+        code += "argument_relative = " + std::to_string(action.relative()) + ";\n";

```

Comment:
Not a compelling reason not to pre-concatenate the string.
```suggestion
        code += action.relative() ? "argument_relative = true;\n" : "argument_relative = false;\n";
```